<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EducationReadWindow</class>
 <widget class="QDialog" name="EducationReadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Дані стажування</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0">
       <widget class="QLabel" name="l_location">
        <property name="text">
         <string>Телефон</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLineEdit" name="le_email"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="l_phone">
        <property name="text">
         <string>Додаткові умови</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QTextEdit" name="te_description">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_education">
        <property name="text">
         <string>Початок у</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_title"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_for_time_start"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="le_option_condition"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="le_for_time_end"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_city">
        <property name="text">
         <string>Необхідний досвід</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_salary">
        <property name="text">
         <string>Опис</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="le_location"/>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="le_phone"/>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Компанія</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="l_title">
        <property name="text">
         <string>Заголовок</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="l_company">
        <property name="text">
         <string>Е-пошта</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="le_req_experience"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_experience">
        <property name="text">
         <string>Кінець у</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="le_city"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="l_description">
        <property name="text">
         <string>Місто</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="l_email">
        <property name="text">
         <string>Локація</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Потрібна освіта</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QComboBox" name="cb_education"/>
      </item>
      <item row="12" column="2">
       <widget class="QLineEdit" name="le_company"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
